card/lib/card/format.rb

```diff
@@ -503,9 +503,13 @@ module Location
       # TESTME
       def page_path title, opts={}
         Rails.logger.warn "Pass only Card::Name to page_path #{title.class}, #{title}" unless Card::Name===title
-        format = opts[:format] ? ".#{opts.delete(:format)}"  : ''
-        query  = opts.present? ? "?#{opts.to_param}"         : ''
-        card_path "#{title.to_name.url_key}#{format}#{query}"
+        card_path page_path_segment(title, opts)
+      end
+      
+      def page_update_path title, opts={}
+        Rails.logger.warn "Pass only Card::Name to page_update_path #{title.class}, #{title}" unless Card::Name===title
+        update_opts = opts[:card] ? opts : { :card => opts }
+        card_path "update/#{page_path_segment(title, update_opts)}"
```

Comment:
Is this really necessary (especially for one call)?.   Can you do :path_opts=>{:action=>:update} and use page_path?